terraform/testcases/xrayreceiver_mock/otconfig.tpl

```diff
@@ -0,0 +1,20 @@
+receivers:
+  awsxray:
+    endpoint: 0.0.0.0:${udp_port}
+    transport: udp
+
+exporters:
+  logging:
+    loglevel: debug
+  awsxray:
+    region: ${region}
+    local_mode: true
+    no_verify_ssl: false
+    endpoint: "${mock_endpoint}"
+
+service:
+  extensions:
+  pipelines:
+    traces:
+      receivers: [awsxray]
+      exporters: [logging, awsxray]
```

Comment:
the same here, copy the same trace pipeline config from the default config.

Reply:
okay